backend/app/routers/review.py

```
@@ -0,0 +1,467 @@
+        return StreamingResponse(
+            buffer_io,
+            media_type="application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+            headers={"Content-Disposition": content_disposition},
+        )
+
+    except FileNotFoundError as e:
+        raise HTTPException(status_code=400, detail=str(e)) from e
+    except Exception as e:
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"导出 Word 失败: {str(e)}",
```

Ruff (0.15.7)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)